ISSUE.md

```diff
@@ -0,0 +1,55 @@
+# Issue reporting on Podman
+
+The Podman team cares for our users and communities.  When someone has a problem with
+Podman and takes the time to report an [issue](https://github.com/containers/podman/issues)
+on our Github, we deeply appreciate the effort.  We want to help. Consider reading
+our [support](SUPPORT.md) document prior to submitting an issue as well.
```

Comment:
The link does not exists yet, I am sure you know it but I mention it anyway so we do not merge it with a 404 link.

Reply:
i missed it in adding new files ... there now./